authors/apps/authentication/serializers.py

```diff
@@ -3,7 +3,20 @@
 from rest_framework import serializers
 
 from .models import User
-
+import re
+
+class CustomValidator:
+    def password_strength(value):
+        """
+        Test if password contains numbers and letters else raise serializer
+        Validation error
+        """
+        if (re.compile(r'^(?=.*[A-Za-z0-9])(?=.*[$@$!%*#?&])[A-Za-z\d$@$!%*#?&]').search(value['password'])) is None:
```

Comment:
This line is too long. Consider refactoring it to:
```suggestion
       if re.compile(
            r'^(?=.*[A-Za-z0-9])(?=.*[$@$!%*#?&])[A-Za-z\d$@$!%*#?&]'
        ).search(value['password']) is None:
```